webserver/static/gulpfile.js

```diff
@@ -113,13 +112,14 @@ function buildScripts() {
   ]).then(writeManifest);
 }
 
-gulp.task('styles', buildStyles);
+gulp.task('styles', function () {
+  return buildStyles(writeManifest);
+});
 gulp.task('scripts', buildScripts);
 
 gulp.task('watch', ['styles', 'scripts'], function () {
-  let watch = require('gulp-watch');
 
-  watch(path.resolve(STATIC_DIR, '**/*.less'), buildStyles);
+  gulp.watch(path.resolve(STATIC_DIR, '**/*.less'), ['styles']);
```

Comment:
This gives output like this now when you save a js file:
```
[15:56:38] Starting 'styles'...
[15:56:38] Finished 'styles' after 3.05 ms
```
just like when you run `gulp styles`